public/422.html

```diff
@@ -1,67 +1,161 @@
-    margin: 0;
-  }
-
-  .rails-default-error-page div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
-
-  .rails-default-error-page div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-
-  .rails-default-error-page h1 {
-    font-size: 100%;
-    color: #730E15;
-    line-height: 1.5em;
-  }
-
-  .rails-default-error-page div.dialog > p {
-    margin: 0 0 1em;
-    padding: 1em;
-    background-color: #F7F7F7;
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #999;
-    border-bottom-left-radius: 4px;
-    border-bottom-right-radius: 4px;
-    border-top-color: #DADADA;
-    color: #666;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-  </style>
-</head>
-
-<body class="rails-default-error-page">
-  <!-- This file lives in public/422.html -->
-  <div class="dialog">
-    <div>
-      <h1>The change you wanted was rejected.</h1>
-      <p>Maybe you tried to change something you didn't have access to.</p>
+  <head>
+    <title>The change you wanted was rejected (422)</title>
+    <style>
+      html { box-sizing: border-box; font-size: 100%; line-height: 1.15; -webkit-text-size-adjust: 100%; }
+      body { margin: 0; }
+      h1 { font-size: 2em; margin: 0.67em 0; }
+      hr { box-sizing: content-box; height: 0; overflow: visible; }
+      a { background-color: transparent; }
+      b, strong { font-weight: bolder; }
+      img { border-style: none; }
+      *, *::before, *::after { box-sizing: inherit; }
+      body { margin: 0; padding: 0; background: #fefefe; font-family: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif; font-weight: normal; line-height: 1.5; color: #0a0a0a; -webkit-font-smoothing: antialiased; -moz-osx-font-smoothing: grayscale; }
+      img { display: inline-block; vertical-align: middle; max-width: 100%; height: auto; -ms-interpolation-mode: bicubic; }
+      .grid-container { padding-right: 0.625rem; padding-left: 0.625rem; max-width: 75rem; margin: 0 auto; }
+      @media print, screen and (min-width: 40em) { .grid-container { padding-right: 0.9375rem; padding-left: 0.9375rem; } }
+      .grid-x { display: flex; flex-flow: row wrap; }
+      .cell { flex: 0 0 auto; min-height: 0px; min-width: 0px; width: 100%; }
+      .grid-x > .small-10, .grid-x > .small-11, .grid-x > .small-12 { flex-basis: auto; }
+      @media print, screen and (min-width: 40em) { .grid-x > .medium-6 { flex-basis: auto; } }
+      .grid-x > .small-10, .grid-x > .small-11, .grid-x > .small-12 { flex: 0 0 auto; }
+      .grid-x > .small-10 { width: 83.3333333333%; }
+      .grid-x > .small-11 { width: 91.6666666667%; }
```

Comment:
shouldn't all the css be minimized? e.g. Could remove `.grid-x > .small-10, .grid-x > .small-11,`, `img`, everything related to lists (ul, li, ol), `.top-bar`, etc.